hometask4/routers/userRouter.js

```diff
@@ -0,0 +1,32 @@
+import express from "express";
+import validator from "../schema/schema.js";
+import { _update, _delete, _create, _autosuggest, _read } from "../controllers/userController.js";
+import { _checkToken } from "../controllers/authController.js";
+import logger from "../logger/logger.js";
+
+const router = express.Router();
+
+router.use((req, res, next) => {
```

Comment:
The app level error handling middleware is already handling errors - no need to duplicate the logic here, you can just leave the "checkToken" middleware at routers level.